libs/natparse/src/main/java/org/amshove/natparse/natural/project/NaturalProjectFileIndexer.java

```diff
@@ -28,49 +29,53 @@ public void indexProject(NaturalProject project)
 			filesystem.streamFilesRecursively(library.getSourcePath())
 				.filter(NaturalFileType::isNaturalFile)
 				.map(this::toNaturalFile)
+				.filter(f -> f.isStructured() && !f.isFailedOnInit())
```

Comment:
This results in DDMs not being indexed (on our site at least), because they don't have a natural header and are therefor not recognized as structured.
Changing this line to
`.filter(f -> (f.isStructured() && !f.isFailedOnInit()) || f.getFiletype() == NaturalFileType.DDM)`
fixed it for me.

Reply:
I think this could be better, as we do not want to filter out anything but Reporting mode
				.filter(f -> (!(f.isReporting() && f.isFailedOnInit())))